elytron/WFLY-20037-19496_promote_oidc_authentication_request_scope_to_community.adoc

```diff
@@ -0,0 +1,119 @@
+:author:            Pedro Silva
+:email:             pesilva@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+OpenID Connect is an authentication mechanism that builds on OAuth 2.0 and allows a user to login to a web application using credentials established by an OpenID provider.
+
+**OIDC whith Additional Scope Values**
+
+WildFly 32 includes the https://docs.wildfly.org/wildfly-proposals/elytron/WFLY-16532-additional-scope-for-auth-request.html[ability to add additional scope values when securing applications using OpenID Connect (OIDC)]. It has now had four releases of bake at preview level and should be promoted to community.
+
+**OIDC whith Request Parameters**
+
+WildFly 33 https://docs.wildfly.org/wildfly-proposals/elytron/WFLY-17143-request-uri-parameters.html[includes the ability to send the request object as a Json Web Token (JWT) when securing an application using OIDC]. The feature also includes the ability to sign and/or encrypt the JWT for added security. It has now had three releases of bake at preview level and should be promoted to community.
+
+=== User Stories
+
+* I'm a developer who wants to use the OpenID Connect to login to my web application using credentials established by an OpenID provider, and I need to use aditional scope values and request rarameters on my application. This is possible using Wildfly with `preview` stability, but I need it at `community` stability.
+
+== Issue Metadata
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-20037
+* https://issues.redhat.com/browse/WFLY-19496
+
+=== Affected Projects or Components
+
+* https://github.com/wildfly/wildfly
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* Managed domain
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* The OIDC request and aditional scope parameters will be available when the server is running at `community` stability.
+
+=== Non-Requirements
+
+== Future Work
+
+== Backwards Compatibility
+
+=== Default Configuration
+
+No incompatibility.
+
+=== Importing Existing Configuration
+
+No incompatibility.
+
+=== Deployments
+
+For standard WildFly, no change in deployment behavior as the basic implementation is unchanged.
+
+=== Interoperability
+
+No incompatibility.
+
+== Implementation Plan
+
+Move
+
+== Admin Clients
+
+None
+
+== Security Considerations
+
+None
+
+== Implementation Plan
+
+* Update Elytron OIDC Client documentation to `community` instead of `preview`;
+* Update Elytron OIDC Subsystem Schema to `community` instead of `preview`;
+* Update all OIDC client source code to `community`instead of `preview`;
+* Update all tests required to OIDC client to `community` instead of `preview`;
+
+[[test_plan]]
+== Test Plan
+
+* WildFly Elytron test suite: Integration test cases implemented to test for functionality.
```

Comment:
Do we have additional testing required in the WildFly own integration test suite?